src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -127,6 +130,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       response.sendRedirect("/login");
       return;
     }
+    
+    if ((user.getMessagesSent() > MAX_MESSAGES) && (user.getAllowMessageDel())) {
+      //  User has sent too many messages 
+      messageStore.deleteOldMessage(user.getId());
+    } 
+    else {
+      user.setMessagesSent(user.getMessagesSent() + 1);
```

Comment:
I think having a incrementMessagesSent method makes more sense because then you don't need to pass in the number.